candid/src/parser/grammar.lalrpop

```diff
@@ -14,7 +14,6 @@ extern {
         "text" => Token::Text(<String>),
         "id" => Token::Id(<String>),
         "null" => Token::Null,
-        "none" => Token::None,
```

Comment:
This is not in your PR description. Which part exactly is removed?

Reply:
Updated the description. `none` was never in the spec. I added it here to support `null` in the option types. Now that we have type annotations from candid, we should be able to use `null` as the spec intended.